plugins/SimulationView/SimulationViewMainComponent.qml

```diff
@@ -0,0 +1,196 @@
+// Copyright (c) 2018 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.4
+import QtQuick.Controls 1.2
+import QtQuick.Layouts 1.1
+import QtQuick.Controls.Styles 1.1
+
+import UM 1.0 as UM
+import Cura 1.0 as Cura
+
+Item
+{
+    property bool is_simulation_playing: false
+    visible: UM.SimulationView.layerActivity && CuraApplication.platformActivity
+
+    PathSlider
+    {
+        id: pathSlider
+        height: UM.Theme.getSize("slider_handle").width
+        width: 250
+
+        anchors.bottom: parent.bottom
+        anchors.bottomMargin: UM.Theme.getSize("default_margin").height
+
+        anchors.horizontalCenter: parent.horizontalCenter
+
+        visible: !UM.SimulationView.compatibilityMode
+
+        // Custom properties
+        handleValue: UM.SimulationView.currentPath
+        maximumValue: UM.SimulationView.numPaths
+
+        // Update values when layer data changes.
+        Connections
+        {
+            target: UM.SimulationView
+            onMaxPathsChanged: pathSlider.setHandleValue(UM.SimulationView.currentPath)
+            onCurrentPathChanged:
+            {
+                // Only pause the simulation when the layer was changed manually, not when the simulation is running
+                if (pathSlider.manuallyChanged)
+                {
+                    playButton.pauseSimulation()
+                }
+                pathSlider.setHandleValue(UM.SimulationView.currentPath)
+            }
+        }
+
+        // Ensure that the slider handlers show the correct value after switching views.
+        Component.onCompleted:
+        {
+            pathSlider.setHandleValue(UM.SimulationView.currentPath)
+        }
+
+    }
+
+    Button
+    {
+        id: playButton
+        iconSource: !is_simulation_playing ? "./resources/simulation_resume.svg": "./resources/simulation_pause.svg"
```

Comment:
I don't know if it's still working in actuality, but these paths look wrong. I'd expect it to have to get to a parent directory first, like `../../resources/simulation_resume.svg`.

However since the SVG files are now only used in this plug-in, I suggest moving the SVG files into the plug-in.

Reply:
They are already in the plugin (in the resources folder)